Events/SearchAndRescueEvent.cs

```diff
@@ -13,24 +14,23 @@ public class SearchAndRescueEvent: Event
 
         static SearchAndRescueEvent()
         {
-    	    VARIABLES.Add("commodity", "The name of the item recovered");
+    	    VARIABLES.Add("commodity", "The commodity recovered");
             VARIABLES.Add("amount", "The amount of the item recovered");
             VARIABLES.Add("reward", "The monetary reward for completing the search and rescue");
         }
 
-        public string commodity => commodityDefinition?.localizedName ?? "unknown commodity";
-
         public int? amount { get; }
 
         public long reward { get; }
 
-        public CommodityDefinition commodityDefinition { get; }
+        [JsonProperty("commodity")]
+        public CommodityDefinition commodity { get; }
```

Comment:
We need a string output here or VA users won't be able to identify which commodity is included in this event.

Reply:
Aha. It will need to have a different name of course. Will "commodityName" do?
Presumably the VAP will need to be changed to suit, but it's inevitable that either that or the Cottle scripts would.

Reply:
No need for changes to the .vap file... the current .vap file is pretty bare bones and doesn't include any specific commands dealing with this event.

Something like `commodityname` would work (the convention is all lowercase). We would of course want to clearly document the change in ChangeLog.md.